paket.lock

```diff
@@ -1,26 +1,25 @@
 NUGET
-  remote: https://nuget.org/api/v2
-  specs:
+  remote: https://www.nuget.org/api/v2
     FAKE (4.10.3)
     FSharp.Compiler.Service (1.4.0.6)
-    FSharp.Formatting (2.12.0)
+    FSharp.Core (4.0.0.1) - redirects: force
+    FSharp.Formatting (2.12)
       FSharp.Compiler.Service (1.4.0.6)
-      FSharpVSPowerTools.Core (2.1.0)
-    FSharpVSPowerTools.Core (2.1.0)
+      FSharpVSPowerTools.Core (2.1)
+    FSharpVSPowerTools.Core (2.1)
       FSharp.Compiler.Service (>= 1.4.0.6)
-    Microsoft.Bcl (1.1.10)
+    Microsoft.Bcl (1.1.10) - framework: net10, net11, net20, net30, net35, net40, net40-full
       Microsoft.Bcl.Build (>= 1.0.14)
-    Microsoft.Bcl.Build (1.0.21) - import_targets: false
-    Microsoft.Net.Http (2.2.29)
+    Microsoft.Bcl.Build (1.0.21) - import_targets: false, framework: net10, net11, net20, net30, net35, net40, net40-full
+    Microsoft.Net.Http (2.2.29) - framework: net10, net11, net20, net30, net35, net40, net40-full
       Microsoft.Bcl (>= 1.1.10)
       Microsoft.Bcl.Build (>= 1.0.14)
     NUnit (2.6.4)
     NUnit.Runners (2.6.4)
-    Octokit (0.17.0)
-      Microsoft.Net.Http
-    SourceLink.Fake (1.1.0)
+    Octokit (0.21.1)
```

Comment:
@forki no idea, but not fixing it seems to break the doc generation in project scaffold.  No idea why.